packages/react-native-renderer/src/__mocks__/react-native/Libraries/ReactPrivate/RawEventTelemetryEventEmitter.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const RawEventTelemetryEventEmitter = {
+  emitEvent: jest.fn(),
+};
+
+module.exports = RawEventTelemetryEventEmitter;
```

Comment:
I don't think this is a faithful Jest mock because it is not using a default export.

I think you should either use `export default …;` or (if Jest does not support that) `module.exports = {default: …};`.

Reply:
I tried both of these, neither worked fwiw.

Reply:
Did you also change the require expression in the mock below to access `default`? (I am curious what didn't work?)